docker/Readme.md

```diff
@@ -0,0 +1,41 @@
+# Gen Containerization
+
+## Quickstart
+
+### Docker Setup
+Please follow the [official Docker installation instructions](https://docs.docker.com/engine/install/) to setup Docker on your target platform. If you would like to use GPU acceleration, please also install the [NVIDIA Container Toolkit / nvidia-docker](https://github.com/NVIDIA/nvidia-docker).
+
+
+### Building Container
+You can build Docker containers for Gen that either target CPU-only or GPU-enabled machines.
+- Please run `docker build -f docker/GenCPU.dockerfile -t gen:cpu-ubuntu20.04 .` for the CPU image
+- Please run `docker build -f docker/GenGPU.dockerfile -t gen:gpu-ubuntu20.04 .` if you want to use GPU acceleration
+Both commands are supposed to be run from the main Gen.jl folder, not this subfolder.
```

Comment:
Perhaps we should have the "easy version" be to pull a pre-built image from Docker Hub, and have building the container on one's own be the "advanced version"?

Reply:
Definitely, I just lack access rights to do that for you. But I would highly recommend that as well. `docker run -it probcomp/gen:cpu-ubuntu20.04 bash` should just work, absolutely.